backend/app/services/share/base_service.py

```
@@ -301,7 +328,27 @@ def create_share_link(
             return self._share_link_to_response(existing_link)
 
         # Generate new share token
+        logger.info(f"[create_share_link] No existing link found, generating new token")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)